ToolTip.ios.js

```diff
@@ -24,15 +24,19 @@ var propTypes = {
 var ViewClass = React.createClass({
   getDefaultProps: function() {
     return {
-      arrowDirection: 'down'
+      arrowDirection: 'down',
+      onHide: () => true,
+      onShow: () => true
     };
   },
 
   showMenu: function() {
     ToolTipMenu.show(findNodeHandle(this.refs.toolTipText), this.getOptionTexts(), this.props.arrowDirection);
+    this.props.onShow();
   },
   hideMenu: function() {
     ToolTipMenu.hide();
+    this.props.onHide();
```

Comment:
wondering if this would create a cycle when invoking hideMenu programmatically? something like ->
` hideMenu() => tooltipMenu.hide => onHide => onBlur will be called => hideMenu ....`